src/components/FinishedList.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+import { ListContainer, ScrollBox } from './Container';
+import TodoItem from './TodoItem';
+import { TodoItemData } from '../interface';
+
+interface Props {
+  finishedList: TodoItemData[];
+  deleteFinishedList: (index: number) => void;
+  addTodoList: (data: string) => void;
+}
```

Comment:
`interface`를 정의할때는 보통 접두어 `I`를 붙여 인터페이스임을 알 수 있게 하는게 일반적입니다~ 그리고 이름이 너무 일반명사인 감이 있죠??
```suggestion
interface IFinishedListProps {
  finishedList: TodoItemData[];
  deleteFinishedList: (index: number) => void;
  addTodoList: (data: string) => void;
}
```

Reply:
Interface라는 개념을 처음 봤는데, 신기하네요.
다음 주차에 Typescript 미션 수행하면서 몇 번 더 여기 와서 참고하고 가게 될 것 같아요.